rest-service/manager_rest/blueprints_manager.py

```diff
@@ -315,6 +321,29 @@ def execute_workflow(self, deployment_id, workflow_id,
 
         return new_execution
 
+    def _check_for_any_active_executions(self):
+        executions = [
+            e.id
+            for e in self.executions_list(is_include_system_workflows=True)
+            if e.status not in e.END_STATES
+        ]
+
+        if executions:
+            raise manager_exceptions.ExistingRunningExecutionError(
+                'You cannot start system wide execution if there are '
```

Comment:
You cannot start a system-wide execution if there are other executions running. Currently running executions: {0}